### ISSUE TITLE
Enhancement Wishlist ...

### ISSUE THREAD

**Issue by mdpradeep** [JuliaComputing/JuliaDT#2]
Here is a list of features that would be good to have in julia eclipse plugin ...
- [x] JDK 1.7 support
- [ ] Auto Complete feature
- [ ] Navigation feature
- [ ] pre-configured julia interpreter (say, based on an environment variable ?)
- [x] Having Julia REPL inside eclipse
- [x] Plotting
- [ ] debugger

**Comment by ViralBShah** [comment]
Any updates here?

**Comment by timholy** [comment]
Obviously, having IDE access to Gallium would be great advertisement for the 0.5 release.

**Comment by ViralBShah** [comment]
The plan is to have debugger support in both Eclipse and Juno for the 0.5 release or soon after. We are waiting on a bit more API shaping and stabilization.

**Comment by timholy** [comment]
> We are waiting on a bit more API shaping and stabilization.

Certainly Base's inner guts are evolving rapidly, and that makes it hard for Gallium to keep up.

**Comment by zhangliye** [comment]
Without an efficient developing tool with a debugger, it is extremely difficult to use Julia in large project.  I love Julia language very much. However, I am starting to thinking about using Python again after using Julia for about one year. I encountered many problems as mentioned in the blog of Philip I. Thomas.I think many Julia users are facing these problems. I hope these problems could be solved before many users leave Julia. 

"Retro on the Julia Programming Language"
https://blog.staffjoy.com/retro-on-the-julia-programming-language-7655121ea341#.nilmra4vg
